db/migrate/20191126071704_add_bmc_to_subnet.rb

```diff
@@ -0,0 +1,5 @@
+class AddBmcToSubnet < ActiveRecord::Migration[5.2]
+  def change
+    add_column :subnets, :bmc_id, :integer
```

Comment:
Should this be a foreign key?

Reply:
Rails won't allow you to reference a non-existent table. Column (`bmc_id`) and references table (`smart-proxies`) must be the same thing which is not the case. Also other columns are not foreign key, I would not bother.